scripts/generate/_test_supported_builds.sh

```diff
@@ -21,24 +21,6 @@ IGNORE_MACOS=0
 RELATIVE_CRATE_PATH=""
 CRATE_TEST_SCRIPT="${SCRIPT_DIR}/_crate_test_build.sh"
 
-while getopts c:f option; do
-  case $option in
-  c)
-    RELATIVE_CRATE_PATH="${OPTARG}"
-    ;;
-  f)
-    GENERATE_FIPS=1
-    ;;
-  m)
-    IGNORE_MACOS=1
-    ;;
-  ?)
-    usage
-    exit 1
-    ;;
-  esac
-done
```

Comment:
I'd like to keep this concept, the reason I wrote it like this was to make it easier to allow us to test individual parts of the process. Otherwise there is a some top-level script that sets a bunch of environment variables, and then it is unclear what the individual scripts actually need / use. Thus if you just want to invoke this script to test changes, you need to know what combination of environment variables need to be set. In the previous design a script could be explicit about the options it expects to be passed in.